### DISCUSSION TITLE
Next.js and Link confirmation

### DISCUSSION THREAD

**Discussion by BMCwebdev** [adobe/react-spectrum#6285] [Q&A] [answered]
Hey there. I've read what seems to be the most relative links on this subject:
https://github.com/adobe/react-spectrum/issues/5476#issuecomment-1826203368
https://github.com/adobe/react-spectrum/pull/5864

Can anyone confirm for me if the MVP features of Next.js Link component can be passed to React Aria Component's Link? That is, `href, replace, scroll, and prefetch`? Can you feed all of these simply to the RouterProvider like so?
```
<RouterProvider
  navigate={router.push}
  href={router.href}
  replace={router.replace}
  scroll={router.scrollTo}
  prefetch={router.prefetch}
>
```
Not sure how prefetch would actually work here, I think you need to pass the href to it. Anyways, perhaps someone here has already done this and could help me along? Thank you!

**Comment by snowystinger** [comment, marked as answer]
Based on the comment in 5476 prefetch is currently impossible
Otherwise this maybe what you're looking for https://react-spectrum.adobe.com/react-aria/routing.html#router-options

**Reply by BMCwebdev** [reply]
Thank you! Yes I am using the Router now (great documentation), and if someone needs prefetch they can use the Next Link.
